src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -400,14 +460,37 @@ const WorkflowView = _.flow(
     }
   }
 
+  describeSelectionModel() {
+    const { modifiedConfig: { rootEntityType }, entityMetadata, entitySelectionModel: { newSetName, selectedEntities, type } } = this.state
+    const { name } = selectedEntities // entityType?
```

Comment:
rogue comment leftover?

Reply:
Yes, this is another thing that I didn't want to forget about. The terms `entityType` and `rootEntityType` are a little confusing in code. `rootEntityType` is what is saved in the method config. `entityType` is the type of object that will be submitted with each job. These are often the same. If I'm remembering this correctly, they are only different if `rootEntityType` is a set type _and_ an entity expression is provided (which is manual in FC while Terra tries to do it automatically).

It's possible that this function is working in all of these cases as-is, but the comment is reminding me that I should go back and make sure that's true.

Reply:
Correction to above: `rootEntityType` is both what is saved in the method config _and_ the object type submitted with each job. `entityType` reflects the type that the user selected from the dropdown. Since set types are resolved to the root entity type when submitted, `rootEntityType` is what is relevant when describing the selection model so `entityType` is not needed here.